images/flaky-test-reporter/Dockerfile

```diff
@@ -25,6 +25,7 @@ ADD . $TEMP_REPO_DIR
 # Build flaky test reporter tool in the container
 RUN make -C $TEMP_REPO_DIR/tools/$TOOL_NAME/
 RUN cp $TEMP_REPO_DIR/tools/$TOOL_NAME/$TOOL_NAME /$TOOL_NAME
+RUN cp $TEMP_REPO_DIR/tools/$TOOL_NAME/config.yaml config.yaml
```

Comment:
why dont we just copy everything under $TOOL_NAME?

Reply:
That would add source files, the README, and other things right? Certainly doable, it adds a bit of bloat though.

Reply:
yes. but its just a few files. Other images add all the files in the repo :)

I am fine either way, but seems like everytime you want something in the image, you will have to keep adding it.